tests/Support/SupportCollectionTest.php

```diff
@@ -1984,6 +1984,128 @@ public function testSortDesc($collection)
         $this->assertEquals(['T10', 'T2', 'T1'], $data->values()->all());
     }
 
+    /**
+     * @dataProvider collectionClassProvider
+     */
```

Comment:
Use attribute instead of annotation for future compatibility.

```php
use PHPUnit\Framework\Attributes\DataProvider
```

```diff
-   /**
-    * @dataProvider collectionClassProvider
-    */
+   #[DataProvider('collectionClassProvider')
```